lib/Contentful.js

```diff
@@ -152,8 +152,13 @@ class Contentful {
         if (!(entry instanceof Object)) {
           return entry;
         }
+        
+        // Assets are not indexed
+        if (entry.sys.type === 'Asset') return m
```

Comment:
Ah, I guess I had a more narrow use-case in mind. I was using algolia for a blog-style project in which it represented the search. In this case, I wanted the search to match posts, not posts and also any images that are fields for the posts, since that wouldn't really make sense.